cerebrium/endpoints/custom-domains.mdx

```diff
@@ -0,0 +1,209 @@
+- Project-level domains - one domain serves all apps in your project
+- Multiple domains can point to the same project
+- Professional branding with your own domain instead of `*.cerebrium.ai` URLs
+
+## How Custom Domains Work
+
+Custom domains in Cerebrium are **project-level**, meaning:
+
+- **One domain serves all apps** in your project
+- **Multiple domains** can point to the same project (useful for different environments or regions)
+- **Professional branding**: Use your own domain instead of `*.cerebrium.ai` URLs
+- **Region-specific**: Each domain is tied to a specific AWS region for optimal performance
+
+## Getting Started
+
+### Prerequisites
+
+- Access to your domain's DNS settings
+- A Cerebrium project
+
+### Step 1: Create Your Custom Domain
+
+1. Navigate to your project settings in the Cerebrium dashboard
+2. Click on the "Custom Domains" tab
+3. Click "Add Custom Domain"
+4. Enter your domain name and select your target AWS region (choose the region closest to your users for optimal latency)
+5. Click "Create Domain"
+
+### Step 2: Configure DNS Records
+
+After creating the domain, you'll receive DNS configuration instructions. Copy these and create a CNAME record in your DNS provider using the DNS record Cerebrium generates for you.
+
+<Note>
+  You can also find your DNS record details later by clicking "DNS Record" in
+  the Custom Domains list.
+</Note>
+
+<Tabs>
+  <Tab title="Subdomains (Recommended)">
+    **Benefits:**
+    - Easier DNS configuration
+    - Allows other subdomains for different purposes
+
+    ```
+    Type: CNAME
+    Name: api
+    Value: proxy.aws.{region}.cerebrium.ai
+    ```
+
+  </Tab>
+
+  <Tab title="Apex Domains">
+    **Benefits:**
+    - Routes all traffic from your domain root
+    - Best for dedicated API domains
+
+    **Note:** ALIAS records are preferred over CNAME since only one CNAME is allowed per domain
+
+    ```
+    Type: CNAME (or ALIAS if supported)
+    Name: @ (or leave blank)
+    Value: proxy.aws.{region}.cerebrium.ai
+    ```
+
+  </Tab>
+
+  <Tab title="Wildcard Domains">
+    **Benefits:**
+    - Catches all subdomains under your domain
+    - Useful for multi-tenant applications
+
+    **Note:** Requires careful consideration of existing subdomains
+
+    ```
+    Type: CNAME
+    Name: *
+    Value: proxy.aws.{region}.cerebrium.ai
+    ```
+
+  </Tab>
+</Tabs>
+
+<Note>
+  You will be prompted to select a `{region}` when creating your custom domain.
+</Note>
+
+### Step 3: Domain Validation
+
+1. After configuring DNS, return to the Cerebrium dashboard
+2. Cerebrium will automatically attempt to validate your DNS records every 30 minutes for up to 2 days
+3. To trigger an immediate validation attempt, click "Validate Domain" next to your domain, even after the 2 days have elapsed
+4. If validation fails, the dialog will show you the last known error. Reach out to us on our [Discord support channel](https://discord.gg/ATj6USmeE2) if you have questions
```

Comment:
you can remove the reach out to support sentance here. We don't put that in docs really since all support queries should go there